src/sqlparse.yrl

```diff
@@ -398,11 +404,23 @@ extra -> NAME  ';' : {extra, unwrap_bin('$1')}.
 
 sql -> cursor_def                     : '$1'.
 sql -> manipulative_statement         : '$1'.
+sql -> plsql_block                    : '$1'.
 sql -> procedure_call                 : '$1'.
 sql -> schema                         : '$1'.
 sql -> WHENEVER NOT FOUND when_action : {when_not_found, '$4'}.
 sql -> WHENEVER SQLERROR  when_action : {when_sql_err, '$3'}.
 
+plsql_block -> plsql_body : '$1'.
+
+plsql_body -> BEGIN statement_pragma_list END : {'plsql_body', '$2'}.
+
+statement_pragma_list ->                       statement_pragma :         ['$1'].
+statement_pragma_list -> statement_pragma_list statement_pragma : '$1' ++ ['$2'].
+
+statement_pragma -> assign_statement : '$1'.
+
+assign_statement -> parameter ':=' scalar_opt_as_exp ';' : {':=', '$1', '$3'}.

```

Comment:
Instead of
```erlang
assign_statement -> parameter ':=' scalar_opt_as_exp ';' : {':=', '$1', '$3'}.
```
Can it be added to this group as following
https://github.com/K2InformaticsGmbH/sqlparse/blob/cb7df10e5ee8660e2dd319fe9ad8af4b77cbfff8/src/sqlparse.yrl#L1129-L1133
```erlang
scalar_opt_as_exp -> scalar_exp ':=' scalar_exp ';' : {':=', '$1', '$3'}. % New PL/SQL only rule
```
Since the following can work for the LHS (parameter part of the rule)
```erlang
scalar_exp -> atom -> parameter_ref -> parameter
```

Reply:
I wouldn't like this because this is a starter version for a proper PL/SQL block and the statements are defined as follows:

![image](https://user-images.githubusercontent.com/1956149/47781270-9cc82880-dcfd-11e8-9cf4-62d9077a12b3.png)

and also the assignment statement is much more complex:

![image](https://user-images.githubusercontent.com/1956149/47781336-bc5f5100-dcfd-11e8-9571-716fcd2abdae.png)